src/testing/index.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="../../node_modules/@types/jasmine/index.d.ts"/>
+
+import { MockCmsService } from './mock.cms.service';
+import { currentCms } from './components';
+import { matchers } from './matchers';
+
+// custom jasmine matchers
+beforeEach(() => jasmine.addMatchers(matchers));
+
+// normal exports
+export { cit, create, contain, direct, provide, stub, stubPipe } from './components';
```

Comment:
These are the primary helpers for running a component test.  They do a lot of the boilerplate-legwork of angular testing.